UM/TaskManagement/HttpRequestScope.py

```diff
@@ -51,6 +51,10 @@ def __init__(self, base: HttpRequestScope) -> None:
             "Content-Type": "application/json",
             "Accept": "application/json"
         }
+        # JsonDecorator probably _should_ have inherited from DefaultUserAgentScope in the first place.
+        # In order to prevent a break in the API by forcing that, just check if there are extra header specs needed.
```

Comment:
I'm not sure it should actually inherit from `DefaultUserAgentScope`. You may want to use this decorator without the user-agent, or with a different one. Those decorators are, as I understood them, designed to be chained.
I would suggest to either:
* Commit to your suggestion and force the inheritance
* Commit to this new behavior and don't mention the inheritence in the comment

On the other hand, I'm overall surprised by this fix. The `header_dict` values should be applied in the `requestHook` method, both the ones from the base and the instance, so I don't see what good this does. I'm probably missing something.

Reply:
Ah, I missed that, sorry.
I think we should close this PR -- sorry 😓